.github/workflows/test-beta.yml

```diff
@@ -0,0 +1,25 @@
+name: test-beta
+
+on:
+  schedule:
+    - cron: "0 6 * * *"
```

Comment:
Could you add a comment saying this is a daily 06h00 run?

I always have to think about the cron job syntax :)